ArmRobot/Assets/Scripts/DomainRandomization.cs

```diff
@@ -220,5 +221,5 @@ void CreateRandomListLights()
         lightComp.range = 20f;
         return light;
     }
-    
```

Comment:
You don't need to comment out all these methods, you could just comment out where you call them from.